src/plugins/intel_cpu/src/nodes/common/dnnl_executor.h

```diff
@@ -34,6 +34,7 @@ class DnnlExecutor {
         DnnlExecutor() = default;
         Primitive execPrim;
         // key is the port number for the primitive that needs memory reordering
+        std::unordered_map<int, MemoryPtr> inputConsts;
```

Comment:
Unfortunatelly, we can not store the memory pointers inside the executor. The main problem, is that the executor may be shared across different nodes of the same type. And with this implementation there might be situations, where some node will be called with a constant from some different node. And as far as I understend this is a corner stone of the whole PR

Reply:
Exactly, thanks a lot! redesigned!